### PR TITLE
add "player helpers" item to Enemies menu

### PR DESCRIPTION
**Pull request by rfomin**
* Add size 4 thermo

* Fix S_LEVPARM/S_PRGWARN for thermo

woof0000

It does not seem trivial to spawn dogs mid-game.

### CODE REVIEW

File: src/m_menu.c [hunk 1]
```diff
@@ -3488,18 +3498,24 @@ static setup_menu_t* enem_settings[] =
...
   enem1_ghost,
   enem1_fuzz,
-
-  enem1_end
 };
 
 setup_menu_t enem_settings1[] =  // Enemy Settings screen
 {
-  {"Cosmetic", S_SKIP|S_TITLE, m_null, M_X, M_Y},
+  {"Player helpers", S_MBF|S_THERMO|S_THRM_SIZE4|S_LEVWARN|S_ACTION,
```
Comment (ceski-1) [inline src/m_menu.c:3513]: Is "Player Helpers" descriptive enough? If not, some random ideas:
```
Helper Bots
AI Companions
AI Followers
AI Helpers
AI Friends
NPC Companions
```

**Review by ceski-1 — COMMENTED** [review]
Seems like a requirement for a source port named Woof!

File: src/m_menu.c (hunk 1 above)
Comment (rfomin) [reply src/m_menu.c:3513]: I like `AI Friends`, but perhaps we shouldn't use the term `AI` these days 😆

Comment (fabiangreffrath) [reply src/m_menu.c:3513]: Let's please avoid using buzz words in our menu. :wink: Doesn't MBF explicitly call them "helper dogs"?

Comment (fabiangreffrath) [reply src/m_menu.c:3513]: "Number Of Single-Player Helper Dogs" 😮

Comment (rfomin) [reply src/m_menu.c:3513]: So, just "Helper Dogs"?

Comment (fabiangreffrath) [reply src/m_menu.c:3513]: That'd be my choice.

Comment (fabiangreffrath) [reply src/m_menu.c]: Play a barking sound whenever the value changes. 😁

Comment (rfomin) [reply src/m_menu.c]: Done in https://github.com/fabiangreffrath/woof/pull/1425/commits/5e9636cdbf50200f1344466e300208b0a3b59b52

Comment (rfomin) [reply src/m_menu.c]: > Play a barking sound whenever the value changes. 😁

Done in https://github.com/fabiangreffrath/woof/pull/1425/commits/13f68772ab35596557d5fe024354da1935383c87

**Pull request merged**